docs/src/DataStructures/SignedRequest.md

```diff
@@ -46,6 +46,12 @@
         }
       ]
     }
-  ]
+  ],
+  "applicationContext": {
+    "url": "https://example.org/myapp/siwf-manifest.json",
+    "hash": [
+      "bciqmdvmxd54zve5kifycgsdtoahs5ecf4hal2ts3eexkgocyc5oca2y"
+    ]
```

Comment:
Why not just have the data here? I don't see a reason to unless we think it is going to get too large.

Reply:
Might get large, e.g. with localization, or in the future if we use the context as a means of allowing the developer to programmatically describe the way their app uses and interacts with user data. Or additional types of assets that may be needed.

Also, if it has a signature from an issuer that adds to size.

Another longer term goal is to power an "app catalog", or allow providers to "list" their apps on their profile. For this to work we would like to have a canonical URL for each app context credential.